run/opencl/ed25519-donna/ed25519-donna-impl-base.h

```diff
@@ -121,6 +121,9 @@ ge25519_scalarmult_base_choose_niels(ge25519_niels *t, uint32_t pos, signed char
 
 /* computes [s]basepoint */
 static void
+#if gpu_nvidia(DEVICE_INFO) && DEV_VER_MAJOR > 500
```

Comment:
The strict `> 500` is a bit weird, meaning 501 or higher. Perhaps you meant `>= 500`.

Reply:
I *think* I had it working with 525.xx so maybe we should use that. Also I now confirmed the 560 or higher CUDA drivers work fine without the noinline, which is great! I'll amend it.